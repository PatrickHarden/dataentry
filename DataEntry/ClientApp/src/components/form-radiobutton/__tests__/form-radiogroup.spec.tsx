import React from 'react';
import { shallow } from 'enzyme';

import FormRadioGroup from '../form-radiogroup';

describe('components', () => {

  describe('form-radiogroup', () => {

    let props:any;

    beforeEach(() => {
        props = {
            name: 'testme',
            label: 'Turn up the radio!',
            options: [{value: 'one', label: 'meow'}, {value: 'two', label: 'bark'}],
            error: false,
            changeHandler: jest.fn()
        }
    });

    // snapshot render
    it('renders according to snapshot', () => {
        expect(shallow(<FormRadioGroup {...props} />)).toMatchSnapshot();
    });
  });
});