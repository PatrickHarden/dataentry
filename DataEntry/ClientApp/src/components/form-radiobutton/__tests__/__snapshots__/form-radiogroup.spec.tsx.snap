// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components form-radiogroup renders according to snapshot 1`] = `
<styled.div
  id="RGtestme"
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMouseLeave={[Function]}
  tabIndex={0}
>
  <FormFieldLabel
    error={false}
    label="Turn up the radio!"
  />
  <FormRadioButton
    changeHandler={[Function]}
    error={false}
    forceFocus={false}
    key="rbkeyone"
    label="meow"
    name="testme"
    optionVal="one"
  />
  <FormRadioButton
    changeHandler={[Function]}
    error={false}
    forceFocus={false}
    key="rbkeytwo"
    label="bark"
    name="testme"
    optionVal="two"
  />
</styled.div>
`;
